docs/using-workflows/adding-secrets.md

```diff
@@ -1,6 +1,9 @@
 # Adding secrets
+#### Encrypted secrets allow you to store sensitive information in your workflow.
```

Comment:
if this is a formatting thing rather than a H4 use `**` ; headers become anchors and are indexed as Table of contents elements, so they should only be structural elements